bundles/org.openhab.binding.pentair/src/main/java/org/openhab/binding/pentair/internal/handler/PentairIntelliFloHandler.java

```diff
@@ -62,7 +62,7 @@ public void dispose() {
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
         if (command instanceof RefreshType) {
-            logger.debug("Intellflo received refresh command");
+            logger.debug("Intellifio received refresh command");
```

Comment:
Why "fio" rather than "flo"? The class is called PentairIntelliFloHandler? See https://www.pentair.com/en-us/products/residential/pool-spa-equipment/pool-pumps/intelliflo_variablespeedhighperfomancepump.html

```suggestion
            logger.debug("IntelliFlo received refresh command");
```